### PR TITLE
add radiospectra (sunpy)

### PR DESCRIPTION
**Pull request by dpshelio**
see https://pypi.org/project/radiospectra

### CODE REVIEW

File: recipes/radiospectra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - matplotlib
+
+test:
+  imports:
+    - radiospectra
+    - radiospectra.sources
+    - radiospectra.tests
+about:
+  home: http://sunpy.org
+  license: BSD-3-Clause
```
Comment (CJ-Wright) [inline recipes/radiospectra/meta.yaml]: Needs a license_family and a license_file

Comment (dpshelio) [reply recipes/radiospectra/meta.yaml]: I've got troubles because the license_file is a `rst`. I'm changing that on pypi first.

**Comment by dpshelio** [comment]
@CJ-Wright  - I still get an error that the LICENSE file cannot be copied. Am I doing something wrong?

**Comment by CJ-Wright** [comment]
Is the file included in the pypi package? (via the manifest file?)

File: recipes/radiospectra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
```
Comment (CJ-Wright) [inline recipes/radiospectra/meta.yaml]: Can this use pip? https://conda-forge.org/docs/meta.html#use-pip

Comment (dpshelio) [reply recipes/radiospectra/meta.yaml]: No, I don't think it can... (or at least I don't know how and with pip it failed). This package uses a submodule ([astropy-helpers](https://github.com/astropy/astropy-helpers)) that implements the `--no-git` argument so it doesn't try to find whether there's an update of itself. I've reverted the change.
@bsipocz - am I wrong?

Comment (bsipocz) [reply recipes/radiospectra/meta.yaml]: afaik you need all your dependencies being available on conda/conda-forge, pip install is not an option (the helpers is not a dependency though as the package includes it as a submodule).

Comment (CJ-Wright) [reply recipes/radiospectra/meta.yaml]: It seems that `astropy-helpers` is a dependency, looking at the circle output.

Comment (bsipocz) [reply recipes/radiospectra/meta.yaml]: I can't see any helpers related output in the circle log, and I don't have time to look into the particulars. In the usual setup `astropy-helpers` should be a submodule and definitely not an installable dependency. Maybe @Cadair can join in to clear up the misunderstandings.

Comment (CJ-Wright) [reply recipes/radiospectra/meta.yaml]: In the travis log https://travis-ci.org/conda-forge/staged-recipes/builds/345783282#L339

Comment (Cadair) [reply recipes/radiospectra/meta.yaml]: As far as the pypi tarball / conda is concerned astropy-helpers should be a bunch of static files. The extra arguments to setup.py stop it running any of the code to update those files from git or pypi.

I don't know the fine details of using pip though, I wonder if @mwcraig could enlighten us all?

Comment (mwcraig) [reply recipes/radiospectra/meta.yaml]: The issue is that the first thing that `astropy_helpers` does when it is run is try to update itself...so it isn't a dependency, per se, but it does try to download during the install, which conda catches in the same way it would catch an attempt to download a dependency.

The `--offline` argument to `setup.py` turns off that behavior. I don't know if such an option is available when installing via pip.

So long story short, I'd stick with `setup.py` for now with this package and others that use `astropy_helpers`.

File: recipes/radiospectra/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sunpy >=0.8
```
Comment (CJ-Wright) [inline recipes/radiospectra/meta.yaml]: Is this a build dep?

Comment (dpshelio) [reply recipes/radiospectra/meta.yaml]: No, you are correct! I missed that one!!

**Comment by CJ-Wright** [comment]
LGTM

File: recipes/radiospectra/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - numpy
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - radiospectra
+    - radiospectra.sources
+    - radiospectra.tests
+about:
```
Comment (CJ-Wright) [inline recipes/radiospectra/meta.yaml]: When you get a chance I'd put a newline here

**Comment by CJ-Wright** [comment]
@dpshelio thank you very much! Welcome to Conda-Forge!

File: recipes/radiospectra/meta.yaml (hunk 4 above)
Comment (dpshelio) [reply recipes/radiospectra/meta.yaml]: Oh! sure!! Thanks!!!
